docs/cli.rst

```diff
@@ -48,6 +51,12 @@ Or with a filename::
     export FLASK_APP=/path/to/hello.py
     flask run
 
+If you use Flask 0.13 or later you can also set the environment variables
+in a ``.env`` file.  For instance like this::
+
+    FLASK_APP=/path/to/hello.py
+    FLASK_DEBUG=1
+
```

Comment:
@mitsuhiko the `.env` file doesn't live anywhere. Some projects include a `.env.template` or similar file that you can use as a template to write your own. You are supposed to write one in each installation of your app. I've seen people write these files manually, even on production deploys. I've also seen them generated by fabric/ansible/etc.

@ThiefMaster Even with a custom command, you need to know where the app is, right? I don't see how this relates to selecting a config file via environment. All the config file stuff happens after the app exists, so nothing you put in the config file will help you locate the app.